### PR TITLE
TS-8

### PR DESCRIPTION
**Pull request by aliceyzhou**
I'm not sure how to make it so it searches through all the objects, currently it only checks if the object passed to it matches the criteria.

### CODE REVIEW

File: trojanscheduler/views.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        section = query[query.index(" ") + 1 :]
+        if object.class_code == code and object.class_section == section:
+            return object
+
+    # Create another view that returns a list of basicclassinfo objects based on class code
+    # If the query passes in is "CSCI" expect to see all basicclassinfo that contains "CSCI" so it'll have CSCI 102, CSCI 103, ...
+
+    def class_code_search(object, code):
+        if object.class_code.equals(code):
+            return object.class_code + " " + object.class_section
```
Comment (markkyaw) [inline trojanscheduler/views.py]: Sorry for the late feedback. Refer to https://github.com/markkyaw/trojan_scheduler/blob/main/api/v1/classes/views/views.py to get an idea on how `view` files should be formatted.
Basically, you'll need to create corresponding `urls.py` as shown in https://github.com/markkyaw/trojan_scheduler/blob/main/api/v1/classes/urls.py
Create a new `url` and call it like `class_search` or something. And the view file will look like the class based view listed here
https://www.django-rest-framework.org/tutorial/3-class-based-views/
Let me know if you wanna pair program on this :)

**Review by markkyaw — CHANGES_REQUESTED** [review]

**Comment by markkyaw** [comment]
Also, add `.DS_Store` to your `.gitignore` file since these are files auto created by macs and we don't need them :)

File: trojanscheduler/views.py (hunk 1 above)
Comment (markkyaw) [reply trojanscheduler/views.py]: To make it search, you'll be doing
`BasicClassSection.objects.filter`
You can read more about them here
https://docs.djangoproject.com/en/4.0/topics/db/queries/

**Comment by aliceyzhou** [comment]
Read the page on querying, edited the view to call filter() and order_by(). Also added the two functions to urls. Overall, I'm not

**Comment by markkyaw** [comment]
[TS-8: Create Views that will return stuff](https://trello.com/c/CeUrWfn9/8-ts-8-create-views-that-will-return-stuff)

**Review by markkyaw — APPROVED** [review]

**Pull request merged**